src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -110,6 +116,25 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
             $this->verboseOutput[$group] = $status;
         }
 
+        if ($ignoreFile) {
+            if (!is_file($ignoreFile)) {
+                throw new \InvalidArgumentException(sprintf('The ignoreFile "%s" does not exist.', $ignoreFile));
+            }
+            set_error_handler(static function ($t, $m) use ($ignoreFile, &$line) {
+                throw new \RuntimeException(sprintf('Invalid pattern found in "%s" on line "%d"', $ignoreFile, 1 + $line).substr($m, 12));
+            });
+            try {
+                foreach (file($ignoreFile) as $line => $pattern) {
+                    if ('#' !== trim($line)[0] ?? '#') {
```

Comment:
No need to use json for the ignoreFile. I've updated the PR to read one pattern per line, ignoring lines that start with a `#`

Reply:
Missing brackets around the ?? operands I suppose